UploadDaemon/Configuration/GlobPatternList.cs

```diff
@@ -77,15 +77,10 @@ public Boolean Matches(string text)
         }
 
         /// <inheritdoc/>
-        public override bool Equals(object obj)
-        {
-            return this.Describe().Equals((obj as GlobPatternList)?.Describe());
-        }
+        public override bool Equals(object other) => other is GlobPatternList gpl && gpl.includePatterns.SequenceEqual(includePatterns)
```

Comment:
Please don't use abbreviations for variables (`gpl`) how about `list` or `otherList` instead?